Nos/Views/ReplyCountLabel.swift

```diff
@@ -0,0 +1,154 @@
+            NSSortDescriptor(keyPath: \Event.identifier, ascending: true)
+        ]
+
+        fetchRequest.predicate = predicate
+        fetchRequest.relationshipKeyPathsForPrefetching = ["author"]
+        _replies = FetchRequest(fetchRequest: fetchRequest)
+    }
+
+    private var isBeingDiscussed: Bool {
+        !replies.isEmpty
+    }
+
+    @MainActor
+    private func computeAvatars() async {
+        guard let socialGraph = currentUser.socialGraph else {
+            return
+        }
+        var followsInDiscussion = Set<Author>()
+        var iterator = replies.makeIterator()
+        while followsInDiscussion.count < 4, let reply = iterator.next() {
+            let author = reply.author
+            if let author, await socialGraph.follows(author.hexadecimalPublicKey) {
+                followsInDiscussion.insert(author)
+            }
+        }
+        avatars = followsInDiscussion.map { $0.profilePhotoURL }
+    }
+
+    private var attributedReplies: AttributedString? {
+        guard isBeingDiscussed else {
+            return nil
+        }
+        switch replyCount {
+        case .discussion:
+            if avatars.isEmpty {
+                return AttributedString(
+                    String(localized: .reply.joinTheDiscussion),
+                    attributes: AttributeContainer(
+                        [NSAttributedString.Key.foregroundColor: UIColor.primaryTxt]
+                    )
+                )
+            } else {
+                return AttributedString(String(localized: .reply.inDiscussion))
+            }
+        case .count:
+            let count = replies.count
+            let string = String(localized: .reply.replies(count))
+            do {
+                var attributed = try AttributedString(markdown: string)
+                if let range = attributed.range(of: "\(count)") {
+                    attributed[range].foregroundColor = .primaryTxt
+                }
+                return attributed
+            } catch {
+                return nil
+            }
+        }
+    }
+
+    var body: some View {
+        HStack(spacing: 0) {
+            StackedAvatarsView(
+                avatarUrls: avatars,
+                size: 20,
+                border: 0
+            )
+            .padding(.trailing, 8)
+
+            if let replies = attributedReplies {
+                Text(replies)
+                    .font(.clarity(.medium, textStyle: .subheadline))
+                    .foregroundColor(Color.secondaryTxt)
+                    .lineLimit(1)
+            }
+        }
+        .onAppear {
+            subscribeToReplies()
+        }
+        .onDisappear {
+            relaySubscriptions.removeAll()
+        }
+        .task {
+            await computeAvatars()
+        }
+        .onChange(of: replies.count) {
+            Task {
+                await computeAvatars()
+            }
+        }
+    }
+
+    func subscribeToReplies() {
+        Task(priority: .background) {
```

Comment:
I think we should bump this up to `.userInitiated`. I've seen the OS hold off on scheduling `.background` priority tasks for several seconds or even minutes if the CPU is constantly busy. In this case we are loading data for the UI in response to the user scrolling so it makes sense to fire these off after the CPU is done handling tasks at the highest priority level.